recipe/meta.yaml

```
@@ -14,10 +14,10 @@ package:
 
 source:
   url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_sanitizer_api/{{ platform }}/cuda_sanitizer_api-{{ platform }}-{{ version }}-archive.{{ extension }}
-  sha256: 1d02913ab7a1389a13b4f66479be73fe6391df29a53123cf86efba5f88e991d2  # [linux64]
-  sha256: 5f37b3d901403094c68b2bdbc34381f3daaae7504fc92e5e8ab8feefe196f2b2  # [ppc64le]
-  sha256: a16ca8a7283b4eb074ee878fc44c85dab8946ed6d2ff6a8893c409ebff3c8e3e  # [aarch64]
-  sha256: 1c1d84647d331759bbed4ef820e3ad26866af7383ddcc0ba02d695b9886d7820  # [win]
+  sha256: e65b92b71467efc156bbc3d1ebaf5b14dfbc4e9f269ab2995a37bbf84530666c  # [linux64]
+  sha256: 869d2dc46a291686f8c40f3b98232cddd85e4696be0d7e119ce92ec46e330590  # [ppc64le]
+  sha256: c01d0b46a50e50292afcca33a3b768a900500d5b442925452801331ee0a61644  # [aarch64]
+  sha256: 8674b7e74d20ec9dc7566b9fa72a8ff1b4713c400000021d81d10e7497bc86d8  # [win]
 
 build:
   number: 0
```

```suggestion
  number: 0
  binary_relocation: false
```
Fix seg fault seen in 12.3